tests/end-to-end/external-solution-integration/pkg/testsuite/check_counter_pod.go

```diff
@@ -1,6 +1,7 @@
 package testsuite
 
 import (
+	retrygo "github.com/avast/retry-go"
```

Comment:
(nit) `github.com/avast/retry-go` uses the package name `retry`. Why do we want to rename this to the imho uglier `retrygo`?

Reply:
To have the same import alias like in "internal/http/cloudevent_client.go".